src/components/InputWrapper.tsx

```diff
@@ -0,0 +1,9 @@
+interface InputWrapperProps {
+  children: React.ReactNode;
+}
+
+function InputWrapper({ children }: InputWrapperProps) {
+  return <div className="contact-form-field">{children}</div>;
+}
+
+export default InputWrapper;
```

Comment:
I don't see the benefit or need for this wrapper because it doesn't seem to add anything other than just checking that the children are of `InputWrapperProps`. The previous div was easier to read and follow in my opinion. l would love to hear your thoughts if you think otherwise, maybe I'm missing something.

Reply:
It is just there to provide the same styles for the `textarea` and `input` tags.

Reply:
Yes, please change it back. I prefer it the same way it was since this way is not adding anything. I also think it's easier to follow that way.

Reply:
Made the change @Ngoakor12